.gitignore

```diff
@@ -23,4 +23,5 @@
 hs_err_pid*
 
 .idea
-target
\ No newline at end of file
+target
+master.iml
```

Comment:
again asked for new line.

Reply:
probably we could make it more generic something like
*.iml

I am afraid now it will try to generate "web-ui.iml"

Reply:
Fixed